README.md

```diff
@@ -1,144 +1,194 @@
+# terminus : Pantheon's Command-Line Interface
+ 
+## Status
+[![Build Status](https://travis-ci.org/pantheon-systems/terminus.svg?branch=master)](https://travis-ci.org/pantheon-systems/terminus)
+[![Dependency Status](https://gemnasium.com/pantheon-systems/terminus.svg)](https://gemnasium.com/pantheon-systems/terminus)
 [![Coverage Status](https://coveralls.io/repos/github/pantheon-systems/terminus/badge.svg?branch=master)](https://coveralls.io/github/pantheon-systems/terminus?branch=master)
-
-Terminus is Pantheon's Command Line Interface (CLI), providing equivalent functionality to the Pantheon Web Dashboard and easier scripting.
+ 
+## About
+Terminus is Pantheon's Command Line Interface (CLI), providing at least equivalent functionality to the Pantheon's
+browser-based Dashboard and easier scripting.
 
 If you would like to contribute, pull requests are welcome!
-
-
-Installation
-------------
-
-**Requirements:**
+ 
+## Dependencies
+### Required
+- A command-line client
 - PHP version 5.5.9 or later
 - [PHP-CLI](http://www.php-cli.com/)
 - [PHP-CURL](http://php.net/manual/en/curl.setup.php)
-- [Composer](https://getcomposer.org/doc/00-intro.md)
 
 Once you have at least the requirements installed, you can install Terminus via Composer or Git. Additionally, you may want to install the optional software below to enhance your use of Terminus:
 
-**Optional but recommended:**
+### Recommended
+- [Composer](https://getcomposer.org/doc/00-intro.md)
 - [Drush](http://docs.drush.org/en/master/install/) (Useful to run incompatible-with-Terminus Drush commands)
 - [WP-CLI](http://wp-cli.org/) (Useful to run incompatible-with-Terminus WP-CLI commands)
 - [Git](https://help.github.com/articles/set-up-git/)
 
-> You can install Terminus just about anywhere on your system. In this README, we'll use `/install/location` to stand in for your chosen installation location.
+You can install Terminus just about anywhere on your system. In this README, we'll use `/install/location` to stand in for your chosen installation location.
 
+## Installation
+### Installing via the Terminus installer
+Run this in your Terminal client:
+```bash
+curl -O https://raw.githubusercontent.com/pantheon-systems/terminus-installer/master/builds/installer.phar && php installer.phar
+```
+For more information on installation options, please see the [Terminus Installer README.md file](https://github.com/pantheon-systems/terminus-installer).
 
-####Installing with Composer
-
+### Installing with Composer
 Run this in your terminal client:
 ```bash
-cd $HOME ; mkdir terminus1x ; cd terminus1x ; composer require pantheon-systems/terminus dev-master ; echo $'\nalias terminus1x=$HOME/terminus1x/vendor/bin/terminus'>>$HOME/.bash_profile ; source $HOME/.bash_profile
+composer global require pantheon-systems/terminus
```

Comment:
We should not document `composer global require` as an installation method. This can, and commonly will, fail depending on what else has been installed via `composer global require`.

Presuming that the global Composer bin directory has been added to the $PATH, then the following is a viable alternative:
```
composer global require consolidation/cgr
cgr pantheon-systems/terminus
```